app/services/idv/steps/document_capture_step.rb

```diff
@@ -43,11 +43,13 @@ def native_camera_ab_testing_variables
 
       def acuant_sdk_upgrade_a_b_testing_variables
         bucket = AbTests::ACUANT_SDK.bucket(flow_session[:document_capture_session_uuid])
```

Comment:
I wonder if we could simplify this by changing the logic of the bucket initialization to default to 100% if testing is disabled?

https://github.com/18F/identity-idp/blob/28ac3ed9c33ca9bbed67ed587b499f43efc5b57c/config/initializers/ab_tests.rb#L16-L18